docs/contrib/Logger.md

```diff
@@ -0,0 +1,448 @@
+# Using the Logging Infrastructure
```

Comment:
I think this file should be kept in the OR docs, but add a direct link from ORFS to OR since ORFS also has access to the Tcl message API.